specification/confidentialledger/data-plane/readme.csharp.md

```diff
@@ -0,0 +1,15 @@
+## C
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+```yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 1
+  clear-output-folder: true
+  client-side-validation: false
+  namespace: Microsoft.ConfidentialLedger
```

Comment:
This looks like a track1 namespace.  @JeffreyRichter , Would this fall under the `Azure.Security` namespace group?

Reply:
Good question. I do not think this is closely related to what we usually put under Azure.Security. The list of pre-approved names spaces (for C#) is here: https://azure.github.io/azure-sdk/dotnet_introduction.html. I think it should either be Azure.Storage (since the ledger is stored) or we should create a new namespace. @KrzysztofCwalina should weigh in.

Reply:
It appears that we've been using `Microsoft.ConfidentialLedger` already, as shown in the `resource-manager` side of things which has already been approved and merged. @syhamza maybe this has already been discussed?